modules/docs/arrow-examples/src/test/kotlin/arrow/FpToTheMax.kt

```diff
@@ -22,16 +22,16 @@ interface Console<F> {
 }
 
 class ConsoleInstance<F>(val delay: MonadDefer<F>) : Console<F> {
-    override fun putStrLn(s: String): Kind<F, Unit> = delay { println(s) }
-    override fun getStrLn(): Kind<F, String> = delay { readLine().orEmpty() }
+    override fun putStrLn(s: String): Kind<F, Unit> = delay.delay { println(s) }
+    override fun getStrLn(): Kind<F, String> = delay.delay { readLine().orEmpty() }
```

Comment:
No need to call the instance name delay anymore

Reply:
yeah... but how would you name it?